src/SdkCommon/AppAuthentication/Azure.Services.AppAuthentication/Microsoft.Azure.Services.AppAuthentication.csproj

```diff
@@ -2,29 +2,36 @@
     <PropertyGroup>
         <PackageId>Microsoft.Azure.Services.AppAuthentication</PackageId>
         <Description>Enables a service to authenticate to Azure services using the developer's Azure Active Directory/ Microsoft account during development, and authenticate as itself (using OAuth 2.0 Client Credentials flow) when deployed to Azure.</Description>
-        <Version>1.0.3</Version>
+        <Version>1.2.0-preview</Version>
```

Comment:
Is there a reason this is preview

Reply:
Yes, this will be a preview release for the new functionality in this release.